### DISCUSSION TITLE
Circuits with Assembling Machine are too complicated and inefficient

### DISCUSSION THREAD

**Discussion by DJToxica** [TechReborn/TechReborn#2529] [Ideas]
Hi,

i have been tinkering with the Assembling Maschine lately and hoped for a way to improve my circuit production.
While Electrum is pretty easy to come by silicon plates are really annoying to produce and inefficient to automate.

So either i havent found a way to make it easier or it needs improvement. Because as it stands at the moment, with a automated rubber tree farm and quarrys which create tons of redstone lapis and glowstone dust its easier to user the basic recipes instead of the Assembling Maschine.

How about if you put Quartz and Sand in a alloy smelter you get silicon ingots and create plates from that?

Kind Regards
DJ

**Comment by saltyseadoggo** [comment]
Applied Energistics 2 lets one simply grind quartz into quartz dust and then smelt that into a silicon item. If Tech Reborn implemented something like that, with the silicon item being able to be compressed into the plate, would that be more reasonable, do you think?
